### PR TITLE
Make portal screen reader friendly

### PR DESCRIPTION
**Pull request by imakshayverma**
Fix issue - #37  
Make pages that aren't compatible with Screen Reader mostly because of missing alternative text for images and labels for search bars.

### CODE REVIEW

**Comment by heaven00** [comment]
@akshay2905 file changes ?

**Comment by imakshayverma** [comment]
Empty commit to create a hook for PR. As this would take sometime. Lets do it in phases. I would start with Hompage.

**Comment by heaven00** [comment]
:+1:

File: ckanext/openbudgetsin_theme/templates/header.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
       {% block header_logo %}
         {% if g.site_logo %}
-          <a class="logo" href="{{ h.url('home') }}"><img src="{{ h.url_for_static_or_external(g.site_logo) }}" alt="{{ g.site_title }}" title="{{ g.site_title }}" /></a>
+          <a class="logo" href="{{ h.url('home') }}"><img src="{{ h.url_for_static_or_external(g.site_logo) }}" alt="Open Budgets India header logo" title="{{ g.site_title }}" /></a>
```
Comment (heaven00) [inline ckanext/openbudgetsin_theme/templates/header.html:20]: Why the text here in alt ?

Comment (imakshayverma) [reply ckanext/openbudgetsin_theme/templates/header.html:20]: The purpose of alt and title are different. Thus they aren't suppose to have same content. Also as alt is used highly in screen readers, SEO and for other purposes. Wanted to have a better description for the logo than just "OpenBudgetsIndia" which is the title.

Comment (heaven00) [reply ckanext/openbudgetsin_theme/templates/header.html:20]: okay :+1:

**Review by heaven00 — APPROVED** [review]

**Pull request merged**